commit 1bfb5e7b0901935b9f7f7702a8fd8da8490f1b86
Author: Eric Anholt <eric@anholt.net>
Date:   Mon May 15 09:28:36 2017 -0700

    drm/vc4: Mark the device as active when enabling runtime PM.
    
    Failing to do so meant that we got a resume() callback on first use of
    the device, so we would leak the bin BO that we allocated during
    probe.
    
    Signed-off-by: Eric Anholt <eric@anholt.net>
    Fixes: 553c942f8b2c ("drm/vc4: Allow using more than 256MB of CMA memory.")

diff --git a/drivers/gpu/drm/vc4/vc4_v3d.c b/drivers/gpu/drm/vc4/vc4_v3d.c
index 882ec06225f1..77ed4ef3b8c8 100644
--- a/drivers/gpu/drm/vc4/vc4_v3d.c
+++ b/drivers/gpu/drm/vc4/vc4_v3d.c
@@ -372,6 +372,7 @@ static int vc4_v3d_bind(struct device *dev, struct device *master, void *data)
 		return ret;
 	}
 
+	pm_runtime_set_active(dev);
 	pm_runtime_use_autosuspend(dev);
 	pm_runtime_set_autosuspend_delay(dev, 40); /* a little over 2 frames. */
 	pm_runtime_enable(dev);
